.github/workflows/stale-issue-pr.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 * * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@72afbce2b0dbd1d903bb142cebe2d15dc307ae57
+        with:
+          stale-issue-message: 'This issue has been automatically marked as stale because it has not had recent activity in the last 90 days. It will be closed in 90 days. Thank you for your contributions.'
+          close-issue-message: 'This issue was closed because there has been no follow activity in 90 days. If you feel this was closed in error please provide evidence on the current production app in a new issue or comment in the existing issue to a maintainer. Thank you for your contributions.'
+          stale-issue-label: 'stale'
+          only-issue-labels: 'type-bug'
```

Comment:
```suggestion
          only-issue-labels: 'type-bug, needs-information'
```

Reply:
I think we want to make issues with `needs-reproduction` stale as well.

`only-issue-labels` expects _all_ listed labels to be present on all issues, which would be a problem here if we added `needs-reproduction`. But we should be able to specify both `needs-reproduction` and `needs-information` under `any-of-issue-labels` so that either one is enough to trigger the stalebot

Reply:
@Gudahtt Update made and ready for another review.